index.js

```diff
@@ -13,6 +13,21 @@ module.exports = {
         optionalDependencies: false,
       },
     ],
+    // changes default `assert` value from `both` to `either`
+    'jsx-a11y/label-has-associated-control': [
+      'error',
+      {
+        labelComponents: [],
+        labelAttributes: [],
+        controlComponents: [],
+        assert: 'either',
+        depth: 25,
+      },
+    ],
+    // rule deprecated in favor of jsx-a11y/label-has-associated-control
+    'jsx-a11y/label-has-for': false,
+    'react/destructuring-assignment': false,
+    'react/sort-comp': false,
```

Comment:
I actually kinda like this one. I feel like this is a slippery slope when we differ too much from the Airbnb style guide. Again, the point of using the Airbnb rules was so that we didn't have to have debates on style rules.

Reply:
Okay, I'll revert this one based on the feedback from you and @AlexEvanczuk. There aren't too many errors in `component-library` if we keep this. I'm not sure about ZP.